src/test/kotlin/no/nav/syfo/oppfolgingstilfelle/bit/kafka/syketilfelle/KafkaSyketilfelleBitConsumerSpek.kt

```diff
@@ -228,6 +229,110 @@ class KafkaSyketilfelleBitConsumerSpek : Spek({
                             oppfolgingstilfelle.end shouldBeEqualTo kafkaSyketilfellebitSykmeldingNy.tom
                         }
                     }
+                    it("tilfelleBit should be ignored if avbrutt") {
```

Comment:
Mulig vi burde se litt generelt på å splitte opp consumer-testene og cronjob-testingen her, sånn at man tester consumer (lagrer uprosesserte biter) og cronob (prosesserer biter og lager oppfølgingstilfeller) hver for seg. Men får evt ta det i egen PR.

Reply:
Ja, kan tenke på det og eventuelt gjøre noe med det i en egen PR 👍